parm/config/config.init

```diff
@@ -16,6 +16,7 @@ export GDASINIT_DIR=${UFS_DIR}/util/gdas_init
 
 export CRES_HIRES=$CASE
 export CRES_ENKF=$CASE_ENKF
+export FRAC_ORO="yes"
```

Comment:
What is `FRAC_ORO` and how does it differ from `FRAC_GRID=.true.`?

Reply:
I see it is used in `ufs-utils`.  The question still begs asking, how does it differ from `FRAC_GRID`